.github/workflows/check-new-groups-generation.yml

```diff
@@ -22,6 +22,8 @@ jobs:
           DECENTRALIZED_SUBGRAPH_API_KEY: ${{ secrets.DECENTRALIZED_SUBGRAPH_API_KEY }} 
           GOOGLE_APPLICATION_CREDENTIALS:  ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS }}
           SH_IGNORE_RESOLVING_ERRORS: "false"
+          dune-api-key: ${{ secrets.SH_DUNE_API_KEY }}
```

Comment:
Hey @sampolgar It seems to me there is typo here "dune-api-key" it should be "SH_DUNE_API_KEY"